toolchain/lc-compile/src/grammar.g

```diff
@@ -212,7 +212,6 @@
         Metadata(-> Left) Separator
         PreImportMetadata(-> Right)
         where(Left -> metadata(Position, _, _))
```

Comment:
You can remove this line too - it is fetching the position to use in the warning.